playbooks/install_stack.yaml

```diff
@@ -376,14 +376,6 @@
       owner: stack
       mode: 0644
 
-  - name: Add ceph to enabled services
-    set_fact:
-      service_envs: "{{ service_envs | union(ceph_env) }}"
-    vars:
-      ceph_env:
-        - "/usr/share/openstack-tripleo-heat-templates/environments/{{ ceph_env_base }}/{{ ceph_env_name }}.yaml"
-    when: ceph_enabled
-
```

Comment:
How the ceph-ansible environment is being added for a OSP16 user with `ceph_enabled` set to True? It was handled here before and I think by removing it you assume the user to add it into dev install overrides?